src/VisualStudio/TestUtilities/IntegrationHelper.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Security;
+using System.Text;
+using System.Threading;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities.Interop;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    internal static class IntegrationHelper
+    {
+        public const int DefaultTimeout = 5000;
```

Comment:
Can you add units either to the identifier or in a comment?  I assume milliseconds, but it would be nice if it was explicit.